content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
+The *Payroll* report, *Work Entry Analysis* report, and *Salary Attachment Report* are default
+reports in the *Payroll* app, and are available for all companies, regardless of location.
+
+Beneath the three default reports are all localization-based reports, organized by country, in
+alphabetical order. These reports contain all the various information for the offered benefits and
+local tax laws.
+
+To view all the available reports for the database, including all the localization-specific ones,
+navigate to :menuselection:`Payroll app --> Reporting` to view the available reports. Click on a
+specific report to view it.
 
 .. image:: reporting/reports.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Report dashboard view showing extra reports for Belgium databases.
 
-If a report is unavailable to a company, a user error pops up, stating `You must be logged in to a
-Belgian company to use this feature.`
+If a report is unavailable to a user, an :guilabel:`Invalid Operation` pop-up window appears,
+stating: `You must be logged in to a (country) company to use this feature.`
 
-At the top of each report, click on :guilabel:`Filters` to display the optional filters for the
-specific report. Filters show information that matches the specific filter parameters. For example,
-when selecting the :guilabel:`Last 365 Day Payslip` filter, only payslips for the last 365 days
-appear. All reports have the option to add a custom filter, and each have different default custom
-filter options. Select the parameters, then click the :guilabel:`Apply` button.
+Filters
+=======
 
-.. image:: reporting/custom-filter.png
-   :align: center
-   :alt: Report dashboard view.
+At the top of each report, the default filters are shown inside the :guilabel:`Search` box.
 
-Clicking on :guilabel:`Favorites` displays three options: :guilabel:`Save the current search`,
-:guilabel:`Add the search to the dashboard`, or :guilabel:`Insert the search to a Google
-spreadsheet`.
+Click the :guilabel:`drop-down arrow` in the search box to display the available
+:guilabel:`filters`. Filters show information that matches the specific filter parameters.
 
-.. image:: reporting/favorites.png
-   :align: center
-   :alt: Favorite options for reports.
+.. example::
+   The :guilabel:`Work Entries Analysis` report has two default filters, the :guilabel:`Current
+   month:(Month)(Year)` filter, and the :guilabel:`Validated` filter. The :guilabel:`Payroll` report
+   only has one default filter, the :guilabel:`Last 365 Days Payslip` filter.
+
+   .. image:: reporting/custom-filter.png
+      :align: center
+      :alt: Filters enabled for the Work Entries Analysis report.
+
+All reports can include custom filters or group information by different metrics (employee,
+department, company, etc.). Some reports have the option to compare the current report to the
+previous time period or year (a :guilabel:`Comparison` option).
+
```

Comment:
Is there a reason you didn't simply tackle each of the three Reporting pages in the Payroll app one by one? Not only would that provide a clearer picture for the reader, but it would help organize this doc a great deal.

Reply:
UPDATE: I can see now that you eventually do that, but this lead-in was a tad confusing. it might be worth considering a slightly reordering/reorganizing of this doc to improve readability

Reply:
My original logic was to explain what reports there were, then I thought explaining why they are seeing what they see (filters) would be a good section before diving into the separate reports. But your logic also makes sense- so I moved the filters to the bottom. I hope that's not too confusing for the reader, since I want to keep all the "reports" together- so after the reports seemed like the best option.